### DISCUSSION TITLE
The results of cases "Examples" folder

### DISCUSSION THREAD

**Discussion by Mubin130** [firemodels/cfast#2136] [General]
Dear all,
Please I would like to know how can I get the results of the cases given in the folder "Examples" by the dowloading of the CFAST code. In fact, I would like to compare and verify the results obtained on my computer.
Thanks in advance.
Mubin

**Comment by Mubin130** [comment]
I m interested in the results of the case of "MultiFloor".

**Comment by preneke** [comment]
I have seen your request. I am training today but I should be able to get you the results today or tomorrow if I don't have time today.

**Comment by preneke** [comment]
[Detectors.zip](https://github.com/user-attachments/files/17509287/Detectors.zip)
Here are the files for the detector.in case

**Comment by preneke** [comment]
[Flashover.zip](https://github.com/user-attachments/files/17509300/Flashover.zip)
Here is the Flashover cases.

**Comment by preneke** [comment]
[Large_Building.zip](https://github.com/user-attachments/files/17509386/Large_Building.zip)
Here are the Large_Building cases.

**Comment by preneke** [comment]
[MultiFloor.zip](https://github.com/user-attachments/files/17509393/MultiFloor.zip)
Here are the MultiFloor case

**Comment by preneke** [comment]
[Sensitivity.zip](https://github.com/user-attachments/files/17509401/Sensitivity.zip)
Here are the Sensitivity case

**Comment by preneke** [comment]
[Simple.zip](https://github.com/user-attachments/files/17509415/Simple.zip)
Here are the Simple case

**Comment by preneke** [comment]
[Users_Guide_Example.zip](https://github.com/user-attachments/files/17509419/Users_Guide_Example.zip)
Here is the Users_Guide_Example it includes the SmokeView files.

**Comment by preneke** [comment]
Sorry to have done it all separately. I couldn't do it all together. It was too big a file. I hope this is everything you need.

**Comment by Mubin130** [comment]
Many thanks for your reply.
